cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env bash
 set -exuo pipefail
 
+ARCH=${ARCH:-"x86_64"}
```

Comment:
The default should be obtained by running `uname`, just as is the case for the `create-containerdisk.sh` script.

And any reason why this script expects the architecture name to be passed via the `$ARCH` environment variable as opposed to `$ARCHITECTURE`? I think we should be consistent.